### DISCUSSION TITLE
Implementation on circular geomtery

### DISCUSSION THREAD

**Discussion by makeabhishek** [AmirMardan/PyFWI#6] [General]
Hi, This is a discussion rather than an issue. I want to know if it is possible to implement the code for circular geometry. Is any example available?

Any answers and suggestions would be appreciated.

**Comment by AmirMardan** [comment]
Sources and receivers can be on top of the model or on two sides. We can't have sources and receivers at the bottom of the model. I will consider this for the next updates (I create an issue for this as #7). However, if you have a circular geometry just with one source and one receiver on each side (like the model you sent me earlier) it should work. I invite you to have a look at the [documentation](https://pyfwi.readthedocs.io/en/latest/sub_doc/example.html#Simple-Example). I basically used a circular geometry and you can modify this model to correspond to the geometry you need.

**Reply by makeabhishek** [reply]
Thanks for your answer @AmirMardan. I have transducer all around the circular geometry, similar to medical tomography.
